### ISSUE TITLE
bi can't read spec on master

### ISSUE THREAD

**Issue by elliottneilclark** [batteries-included/batteries-included#934]
```
bix bi start http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec
```

I created an install, that has id: `batt_01922ffacf7b756e94f027acb489f34d`

Running the above command results in:

```
➜  batteries-included git:(master) bix bi -v=debug start http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec
[2024-09-26T15:21:19-0500]: Running bi with args: -v=debug start http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec
3:21PM DBG cmd/start.go:43 Starting Batteries Included Installation installSpec=http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec
3:21PM DBG specs/get_spec.go:47 Reading local file path=/home/elliott/.local/state/bi/installs/http:/home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec/spec.json
3:21PM DBG installs/env.go:101 Didn't find install path=/home/elliott/.local/state/bi/installs/http:/home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec/spec.json source=file
3:21PM DBG installs/env.go:101 Didn't find install path=http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec source=url
Usage:
  bi start [install-slug|install-spec-url|install-spec-file] [flags]

Flags:
  -h, --help   help for start

Global Flags:
  -c, --color              Use color in logs (default true)
      --config string      config file (default is $XDG_CONFIG_HOME/bi/bi.yaml)
  -v, --verbosity string   Log level (debug, info, warn, error (default "WARN")

3:21PM ERR cmd/root.go:39 Error executing command error="error reading install env: no spec found"
➜  batteries-included git:(master) 

```

However curl gets the spec:

```
➜  batteries-included git:(master) curl  http://home.127-0-0-1.batrsinc.co:4100/api/v1/installations/batt_01922ffacf7b756e94f027acb489f34d/spec      

{"jwt":{"protected":"eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9","signature":"bPMlB4Qn_ZUi_e-As49y5EALJac1v_8JPQ_H0rA2MnsAMTCxSQpcoNaOfFJaJ9mOZmT4tjx-vA8wZK2YPNMPAw","payload":"eyJzbHVnIjoicm91bmQtYmlraW5pLWRvbGxhciIsImt1YmVfY2x1c3RlciI6eyJjb25maWciOnt9LCJwcm92aWRlciI6ImtpbmQifSwidGFyZ2V0X3N1bW1hcnkiOnsia25hdGl2ZV9zZXJ2aWNlcyI6W10sIm5vdGVib29rcyI6W10sImJhdHRlcmllcyI6W3siaWQiOiJiYXR0XzAxOTIyZmZmNjRlZTdhOWZhNTc3OTUyMGE1MmQ2ZjE5IiwidHlwZSI6ImlzdGlvIiwiY29uZmlnIjp7InR5cGUiOiJpc3RpbyIsIm5hbWVzcGFjZSI6ImJhdHRlcnktaXN0aW8iLCJwaWxvdF9pbWFnZSI6ImRvY2tlci5pby9pc3Rpby9waWxvdDoxLjIzLjItZGlzdHJvbGVzcyIsIm5hbWVzcGFjZV9vdmVycmlkZSI6bnVsbCwicGlsb3RfaW1hZ2VfbmFtZV9vdmVycmlkZSI6bnVsbCwicGlsb3RfaW1hZ2VfdGFnX292ZXJyaWRlIjpudWxsfSwiZ3JvdXAiOiJuZXRfc2VjIiwiaW5zZXJ0ZWRfYXQiOm51bGwsInVwZGF0ZWRfYXQiOm51bGx9LHsiaWQiOiJiYXR0XzAxOTIyZmZmNjRlZTc4MWU5NDAwMDI4MzYwNDBhOTE0IiwidHlwZSI6ImlzdGlvX2dhdGV3YXkiLCJjb25maWciOnsidHlwZSI6ImlzdGlvX2dhdGV3YXkiLCJwcm94eV9pbWFnZSI6ImRvY2tlci5pby9pc3Rpby9wcm94eXYyOjEuMjMuMi1kaXN0cm9sZXNzIiwicHJveHlfaW1hZ2VfbmFtZV9vdmVycmlkZSI6bnVsbCwicHJveHlfaW1hZ2VfdGFnX292ZXJyaWRlIjpudWxsfSwiZ3JvdXAiOiJuZXRfc2VjIiwiaW5zZXJ0ZWRfYXQiOm51bGwsInVwZGF0ZWRfYXQiOm51bGx9LHsiaWQiOiJiYXR0XzAxOTIyZmZmNjRlZTdkOGViNmNkMWEwNDc1M2I2MDdjIiwidHlwZSI6ImJhdHRlcnlfY29yZSIsImNvbmZpZyI6eyJ0eXBlIjoiYmF0dGVyeV9jb3JlIiwidXNhZ2UiOiJkZXZlbG9wbWVudCIsImNsdXN0ZXJfdHlwZSI6ImtpbmQiLCJjb3JlX25hbWVzcGFjZSI6ImJhdHRlcnktY29yZSIsImJhc2VfbmFtZXNwYWNlIjoiYmF0dGVyeS1iYXNlIiwiZGF0YV9uYW1lc3BhY2UiOiJiYXR0ZXJ5LWRhdGEiLCJhaV9uYW1lc3BhY2UiOiJiYXR0ZXJ5LWFpIiwiZGVmYXVsdF9zaXplIjoidGlueSIsImNsdXN0ZXJfbmFtZSI6InJvdW5kLWJpa2luaS1kb2xsYXIiLCJpbnN0YWxsX2lkIjoiYmF0dF8wMTkyMmZmYWNmN2I3NTZlOTRmMDI3YWNiNDg5ZjM0ZCIsImNvbnRyb2xfandrIjp7ImNydiI6IkVkMjU1MTkiLCJkIjoiT2ZhRVF0eHNPXzVwc291dTlFR2otVk52cDJJUFdCLUg1OUpVOC03dmhpNCIsImt0eSI6Ik9LUCIsIngiOiJGSlVnMWtVZDdwX1dhN2ljd3d5d0hISkFFZ3BGdDhBMVhjYVFzeHR6eEFnIn0sInVwZ3JhZGVfZGF5c19vZl93ZWVrIjpbdHJ1ZSx0cnVlLHRydWUsdHJ1ZSxmYWxzZSxmYWxzZSxmYWxzZV0sInZpcnR1YWxfdXBncmFkZV9kYXlzX29mX3dlZWsiOlsiTW9uZGF5IiwiVHVlc2RheSIsIldlZG5lc2RheSIsIlRodXJzZGF5Il0sInVwZ3JhZGVfc3RhcnRfaG91ciI6MTgsInVwZ3JhZGVfZW5kX2hvdXIiOjIzfSwiZ3JvdXAiOiJtYWdpYyIsImluc2VydGVkX2F0IjpudWxsLCJ1cGRhdGVkX2F0IjpudWxsfSx7ImlkIjoiYmF0dF8wMTkyMmZmZjY0ZWU3MWM3YWUwNTEyNjI5ZDVjZWE4YiIsInR5cGUiOiJtZXRhbGxiIiwiY29uZmlnIjp7InR5cGUiOiJtZXRhbGxiIiwiY29udHJvbGxlcl9pbWFnZSI6InF1YXkuaW8vbWV0YWxsYi9jb250cm9sbGVyOnYwLjE0LjgiLCJzcGVha2VyX2ltYWdlIjoicXVheS5pby9tZXRhbGxiL3NwZWFrZXI6djAuMTQuOCIsImZycm91dGluZ19pbWFnZSI6InF1YXkuaW8vZnJyb3V0aW5nL2Zycjo5LjEuMCIsImVuYWJsZV9wb2RfbW9uaXRvciI6ZmFsc2UsInNwZWFrZXJfaW1hZ2VfbmFtZV9vdmVycmlkZSI6bnVsbCwic3BlYWtlcl9pbWFnZV90YWdfb3ZlcnJpZGUiOm51bGwsImNvbnRyb2xsZXJfaW1hZ2VfbmFtZV9vdmVycmlkZSI6bnVsbCwiY29udHJvbGxlcl9pbWFnZV90YWdfb3ZlcnJpZGUiOm51bGwsImZycm91dGluZ19pbWFnZV9uYW1lX292ZXJyaWRlIjpudWxsLCJmcnJvdXRpbmdfaW1hZ2VfdGFnX292ZXJyaWRlIjpudWxsfSwiZ3JvdXAiOiJuZXRfc2VjIiwiaW5zZXJ0ZWRfYXQiOm51bGwsInVwZGF0ZWRfYXQiOm51bGx9LHsiaWQiOiJiYXR0XzAxOTIyZmZmNjRlZTdlY2VhZThiM2E4NjY0NDI3NmYyIiwidHlwZSI6ImNsb3VkbmF0aXZlX3BnIiwiY29uZmlnIjp7InR5cGUiOiJjbG91ZG5hdGl2ZV9wZyIsImltYWdlIjoiZ2hjci5pby9jbG91ZG5hdGl2ZS1wZy9jbG91ZG5hdGl2ZS1wZzoxLjI0LjAiLCJpbWFnZV90YWdfb3ZlcnJpZGUiOm51bGwsImltYWdlX25hbWVfb3ZlcnJpZGUiOm51bGx9LCJncm91cCI6ImRhdGEiLCJpbnNlcnRlZF9hdCI6bnVsbCwidXBkYXRlZF9hdCI6bnVsbH0seyJpZCI6ImJhdHRfMDE5MjJmZmY2NGVmNzI0Y2JjYzc3MzQwY2MyMzBlZmYiLCJ0eXBlIjoic3RhbGVfcmVzb3VyY2VfY2xlYW5lciIsImNvbmZpZyI6eyJ0eXBlIjoic3RhbGVfcmVzb3VyY2VfY2xlYW5lciIsImRlbGF5Ijo5MDAwMDAsImRlbGF5X292ZXJyaWRlIjpudWxsfSwiZ3JvdXAiOiJtYWdpYyIsImluc2VydGVkX2F0IjpudWxsLCJ1cGRhdGVkX2F0IjpudWxsfV0sInRyYWRpdGlvbmFsX3NlcnZpY2VzIjpbXSwibW9kZWxfaW5zdGFuY2VzIjpbXSwicG9zdGdyZXNfY2x1c3RlcnMiOlt7ImlkIjpudWxsLCJuYW1lIjoiY29udHJvbHNlcnZlciIsInR5cGUiOiJpbnRlcm5hbCIsImRhdGFiYXNlIjp7Im5hbWUiOiJjb250cm9sIiwib3duZXIiOiJiYXR0ZXJ5LWNvbnRyb2wtdXNlciJ9LCJ1c2VycyI6W3sicG9zaXRpb24iOm51bGwsInVzZXJuYW1lIjoiYmF0dGVyeS1jb250cm9sLXVzZXIiLCJyb2xlcyI6WyJjcmVhdGVkYiIsImxvZ2luIl0sImNyZWRlbnRpYWxfbmFtZXNwYWNlcyI6WyJiYXR0ZXJ5LWNvcmUiXX1dLCJzdG9yYWdlX2NsYXNzIjpudWxsLCJpbnNlcnRlZF9hdCI6bnVsbCwidXBkYXRlZF9hdCI6bnVsbCwibnVtX2luc3RhbmNlcyI6MSwidmlydHVhbF9zaXplIjoidGlueSIsInBhc3N3b3JkX3ZlcnNpb25zIjpbeyJ2ZXJzaW9uIjoxLCJ1c2VybmFtZSI6ImJhdHRlcnktY29udHJvbC11c2VyIiwicGFzc3dvcmQiOiJZS0VFUUZISTdBQ1BHNUg2Nk1HSTdVVEEifV0sImNwdV9yZXF1ZXN0ZWQiOjUwMCwiY3B1X2xpbWl0cyI6NTAwLCJtZW1vcnlfcmVxdWVzdGVkIjo1MzY4NzA5MTIsIm1lbW9yeV9saW1pdHMiOjUzNjg3MDkxMiwicHJvamVjdF9pZCI6bnVsbCwic3RvcmFnZV9zaXplIjo1MzY4NzA5MTIsInZpcnR1YWxfc3RvcmFnZV9zaXplX3JhbmdlX3ZhbHVlIjo1MDM1OTMxMTIwfV0sInJlZGlzX2luc3RhbmNlcyI6W10sInByb2plY3RzIjpbXSwiZmVycmV0X3NlcnZpY2VzIjpbXSwiaXBfYWRkcmVzc19wb29scyI6W10sInN0YWJsZV92ZXJzaW9uc19yZXBvcnQiOm51bGwsImNhcHR1cmVkX2F0IjpudWxsLCJpbnN0YWxsX3N0YXR1cyI6bnVsbH0sImluaXRpYWxfcmVzb3VyY2VzIjp7Ii9jbHVzdGVyX3JvbGVfYmluZGluZy9iYXR0ZXJpZXNfaW5jbHVkZWRfYm9vdHN0cmFwIjp7ImFwaVZlcnNpb24iOiJyYmFjLmF1dGhvcml6YXRpb24uazhzLmlvL3YxIiwia2luZCI6IkNsdXN0ZXJSb2xlQmluZGluZyIsIm1ldGFkYXRhIjp7ImFubm90YXRpb25zIjp7ImJhdHRlcnkvaGFzaCI6IllLMkRZRU5JTEZYQ0xHTUFMQ1ZENUFEUUlDRFFVTTNRWEZPN0Q0R0dKTlhTTDI2S0xGUlE9PT09In0sImxhYmVscyI6eyJhcHAiOiJiYXR0ZXJ5LWNvcmUiLCJhcHAua3ViZXJuZXRlcy5pby9tYW5hZ2VkLWJ5IjoiYmF0dGVyaWVzLWluY2x1ZGVkIiwiYXBwLmt1YmVybmV0ZXMuaW8vbmFtZSI6ImJhdHRlcnktY29yZSIsImFwcC5rdWJlcm5ldGVzLmlvL3ZlcnNpb24iOiJsYXRlc3QiLCJiYXR0ZXJ5L2FwcCI6ImJhdHRlcnktY29yZSIsImJhdHRlcnkvbWFuYWdlZCI6InRydWUiLCJiYXR0ZXJ5L21hbmFnZWQuZGlyZWN0IjoidHJ1ZSIsImJhdHRlcnkvb3duZXIiOiJiYXR0XzAxOTIyZmZmNjRlZTdkOGViNmNkMWEwNDc1M2I2MDdjIiwidmVyc2lvbiI6ImxhdGVzdCJ9LCJuYW1lIjoiYmF0dGVyaWVzLWluY2x1ZGVkOmJvb3RzdHJhcCJ9LCJyb2xlUmVmIjp7ImFwaUdyb3VwIjoicmJhYy5hdXRob3JpemF0aW9uLms4cy5pbyIsImtpbmQiOiJDbHVzdGVyUm9sZSIsIm5hbWUiOiJjbHVzdGVyLWFkbWluIn0sInN1YmplY3RzIjpbeyJraW5kIjoiU2VydmljZUFjY291bnQiLCJuYW1lIjoiYm9vdHN0cmFwIiwibmFtZXNwYWNlIjoiYmF0dGVyeS1jb3JlIn1dfSwiL2pvYi9ib290c3RyYXAiOnsiYXBpVmVyc2lvbiI6ImJhdGNoL3YxIiwia2luZCI6IkpvYiIsIm1ldGFkYXRhIjp7ImFubm90YXRpb25zIjp7ImJhdHRlcnkvaGFzaCI6IkVFTEhMMkoyRU40TExBMjRIU05VTjdKQjRBWFZEWUE2UTRXSU80V0FKNVBYRTdYQVdXRFE9PT09In0sImxhYmVscyI6eyJhcHAiOiJiYXR0ZXJ5LWNvcmUiLCJhcHAua3ViZXJuZXRlcy5pby9tYW5hZ2VkLWJ5IjoiYmF0dGVyaWVzLWluY2x1ZGVkIiwiYXBwLmt1YmVybmV0ZXMuaW8vbmFtZSI6ImJhdHRlcnktY29yZSIsImFwcC5rdWJlcm5ldGVzLmlvL3ZlcnNpb24iOiJsYXRlc3QiLCJiYXR0ZXJ5L2FwcCI6ImJhdHRlcnktY29yZSIsImJhdHRlcnkvbWFuYWdlZCI6InRydWUiLCJiYXR0ZXJ5L21hbmFnZWQuZGlyZWN0IjoidHJ1ZSIsImJhdHRlcnkvb3duZXIiOiJiYXR0XzAxOTIyZmZmNjRlZTdkOGViNmNkMWEwNDc1M2I2MDdjIiwic2lkZWNhci5pc3Rpby5pby9pbmplY3QiOiJmYWxzZSIsInZlcnNpb24iOiJsYXRlc3QifSwibmFtZSI6ImJvb3RzdHJhcCIsIm5hbWVzcGFjZSI6ImJhdHRlcnktY29yZSJ9LCJzcGVjIjp7ImJhY2tvZmZMaW1pdCI6NiwiY29tcGxldGlvbnMiOjEsInBhcmFsbGVsaXNtIjoxLCJ0ZW1wbGF0ZSI6eyJtZXRhZGF0YSI6eyJhbm5vdGF0aW9ucyI6e30sImxhYmVscyI6eyJhcHAiOiJiYXR0ZXJ5LWNvcmUiLCJhcHAua3ViZXJuZXRlcy5pby9jb21wb25lbnQiOiJib290c3RyYXAiLCJhcHAua3ViZXJuZXRlcy5pby9tYW5hZ2VkLWJ5IjoiYmF0dGVyaWVzLWluY2x1ZGVkIiwiYXBwLmt1YmVybmV0ZXMuaW8vbmFtZSI6ImJhdHRlcnktY29yZSIsImFwcC5rdWJlcm5ldGVzLmlvL3ZlcnNpb24iOiJsYXRlc3QiLCJiYXR0ZXJ5L2FwcCI6ImJhdHRlcnktY29yZSIsImJhdHRlcnkvY29tcG9uZW50IjoiYm9vdHN0cmFwIiwiYmF0dGVyeS9tYW5hZ2VkIjoidHJ1ZSIsImJhdHRlcnkvbWFuYWdlZC5pbmRpcmVjdCI6InRydWUiLCJiYXR0ZXJ5L293bmVyIjoiYmF0dF8wMTkyMmZmZjY0ZWU3ZDhlYjZjZDFhMDQ3NTNiNjA3YyIsImNvbXBvbmVudCI6ImJvb3RzdHJhcCIsInNpZGVjYXIuaXN0aW8uaW8vaW5qZWN0IjoiZmFsc2UiLCJ2ZXJzaW9uIjoibGF0ZXN0In19LCJzcGVjIjp7ImF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW4iOnRydWUsImNvbnRhaW5lcnMiOlt7ImVudiI6W3sibmFtZSI6IlJFTEVBU0VfQ09PS0lFIiwidmFsdWUiOiJYRzdPU0w3RlJNTlVCUzM3SEs2T0ZKVktLQ1g1T0NLS0RGRldISTRPWjRWMzRYSjY2U1c2NkM1Q04yQUdOSzUzIn0seyJuYW1lIjoiUkVMRUFTRV9ESVNUUklCVVRJT04iLCJ2YWx1ZSI6Im5vbmUifSx7Im5hbWUiOiJCT09UU1RSQVBfU1VNTUFSWV9QQVRIIiwidmFsdWUiOiIvdmFyL3J1bi9zZWNyZXRzL3N1bW1hcnkvc3VtbWFyeS5qc29uIn1dLCJpbWFnZSI6InB1YmxpYy5lY3IuYXdzL2JhdHRlcmllcy1pbmNsdWRlZC9rdWJlLWJvb3RzdHJhcDowLjIwLjAtNjYwMDU1MDEiLCJpbWFnZVB1bGxQb2xpY3kiOiJJZk5vdFByZXNlbnQiLCJuYW1lIjoiYm9vdHN0cmFwIiwidm9sdW1lTW91bnRzIjpbeyJtb3VudFBhdGgiOiIvdmFyL3J1bi9zZWNyZXRzL3N1bW1hcnkiLCJuYW1lIjoic3VtbWFyeSJ9XX1dLCJyZXN0YXJ0UG9saWN5IjoiT25GYWlsdXJlIiwic2VydmljZUFjY291bnQiOiJib290c3RyYXAiLCJzZXJ2aWNlQWNjb3VudE5hbWUiOiJib290c3RyYXAiLCJ0b2xlcmF0aW9ucyI6W3sia2V5IjoiQ3JpdGljYWxBZGRvbnNPbmx5Iiwib3BlcmF0b3IiOiJFeGlzdHMifV0sInZvbHVtZXMiOlt7Im5hbWUiOiJzdW1tYXJ5Iiwic2VjcmV0Ijp7InNlY3JldE5hbWUiOiJpbml0aWFsLXRhcmdldC1zdW1tYXJ5In19XX19fX0sIi9uYW1lc3BhY2UvYmF0dGVyeV9jb3JlIjp7ImFwaVZlcnNpb24iOiJ2MSIsImtpbmQiOiJOYW1lc3BhY2UiLCJtZXRhZGF0YSI6eyJhbm5vdGF0aW9ucyI6eyJiYXR0ZXJ5L2hhc2giOiJNQ0I3T01JUVlNTkVCT0U3RkQ3Q1hSNk1JM1dHVkZHVzNWUUhSUzZRQk5aSE1GNkRXWElRPT09PSJ9LCJsYWJlbHMiOnsiYXBwIjoiYmF0dGVyeS1jb3JlIiwiYXBwLmt1YmVybmV0ZXMuaW8vbWFuYWdlZC1ieSI6ImJhdHRlcmllcy1pbmNsdWRlZCIsImFwcC5rdWJlcm5ldGVzLmlvL25hbWUiOiJiYXR0ZXJ5LWNvcmUiLCJhcHAua3ViZXJuZXRlcy5pby92ZXJzaW9uIjoibGF0ZXN0IiwiYmF0dGVyeS9hcHAiOiJiYXR0ZXJ5LWNvcmUiLCJiYXR0ZXJ5L21hbmFnZWQiOiJ0cnVlIiwiYmF0dGVyeS9tYW5hZ2VkLmRpcmVjdCI6InRydWUiLCJiYXR0ZXJ5L293bmVyIjoiYmF0dF8wMTkyMmZmZjY0ZWU3ZDhlYjZjZDFhMDQ3NTNiNjA3YyIsImlzdGlvLWluamVjdGlvbiI6ImVuYWJsZWQiLCJ2ZXJzaW9uIjoibGF0ZXN0In0sIm5hbWUiOiJiYXR0ZXJ5LWNvcmUifX0sIi9zZXJ2aWNlX2FjY291bnQvYm9vdHN0cmFwIjp7ImFwaVZlcnNpb24iOiJ2MSIsImtpbmQiOiJTZXJ2aWNlQWNjb3VudCIsIm1ldGFkYXRhIjp7ImFubm90YXRpb25zIjp7ImJhdHRlcnkvaGFzaCI6IkFJS0JDQVdOQlNOWkRBRVVNWVpQMlpZRExVVzdYQVpES1hHQjYzV0g1V0pVMzNGSDRTSFE9PT09In0sImxhYmVscyI6eyJhcHAiOiJiYXR0ZXJ5LWNvcmUiLCJhcHAua3ViZXJuZXRlcy5pby9tYW5hZ2VkLWJ5IjoiYmF0dGVyaWVzLWluY2x1ZGVkIiwiYXBwLmt1YmVybmV0ZXMuaW8vbmFtZSI6ImJhdHRlcnktY29yZSIsImFwcC5rdWJlcm5ldGVzLmlvL3ZlcnNpb24iOiJsYXRlc3QiLCJiYXR0ZXJ5L2FwcCI6ImJhdHRlcnktY29yZSIsImJhdHRlcnkvbWFuYWdlZCI6InRydWUiLCJiYXR0ZXJ5L21hbmFnZWQuZGlyZWN0IjoidHJ1ZSIsImJhdHRlcnkvb3duZXIiOiJiYXR0XzAxOTIyZmZmNjRlZTdkOGViNmNkMWEwNDc1M2I2MDdjIiwidmVyc2lvbiI6ImxhdGVzdCJ9LCJuYW1lIjoiYm9vdHN0cmFwIiwibmFtZXNwYWNlIjoiYmF0dGVyeS1jb3JlIn19fX0"}}%    
```

**Comment by JTarasovic** [comment]
Not sure. I can't repro

```
bix bi start -v=debug http://127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec
[2024-09-26T15:30:39-0500]: Running bi with args: start -v=debug http://127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec
3:30PM DBG cmd/start.go:43 Starting Batteries Included Installation installSpec=http://127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec
3:30PM DBG specs/get_spec.go:47 Reading local file path=/home/jason/.local/state/bi/installs/http:/127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec/spec.json
3:30PM DBG installs/env.go:101 Didn't find install path=/home/jason/.local/state/bi/installs/http:/127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec/spec.json source=file
3:30PM DBG installs/env.go:104 Found install path=http://127.0.0.1:4100/api/v1/installations/batt_01922a068f717454ae566d33ddbbf6e1/spec source=url
3:30PM DBG installs/write.go:34 Removing install directory path=/home/jason/.local/state/bi/installs/int-test
3:30PM DBG installs/env.go:30 Initializing install slug=int-test
3:30PM DBG specs/write_summary.go:19 Writing state summary to  path=/home/jason/.local/state/bi/installs/int-test/summary.json
```

**Comment by elliottneilclark** [comment]
Try without the ip. Try using the hostname
